joint_trajectory_controller/test/test_trajectory.cpp

```diff
@@ -62,10 +63,11 @@ TEST(TestTrajectory, initialize_trajectory)
     const auto now = rclcpp::Clock().now();
     auto traj = joint_trajectory_controller::Trajectory(empty_msg);
     const auto traj_starttime = traj.time_from_start();
+    rclcpp::Duration period = rclcpp::Duration::from_nanoseconds(1e7);
 
-    trajectory_msgs::msg::JointTrajectoryPoint expected_point;
+    trajectory_msgs::msg::JointTrajectoryPoint output_point;
```

Comment:
I noticed as well that `expected_point` was named in a funny way. Glad to see it renamed  :+1: